packages/bsky/src/api/app/bsky/purchase/getFeatures.ts

```diff
@@ -0,0 +1,50 @@
+import { Server } from '../../../../lexicon'
+import AppContext from '../../../../context'
+import { Features } from '@atproto/api/dist/client/types/app/bsky/purchase/getFeatures'
```

Comment:
I'm not positive on this, but I think the move here is to reference this type from `packages/bsky/src/lexicon/types/app/bsky/purchase/getFeatures.ts`, which is local to the app view package.